hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -568,6 +561,39 @@ private void checkIterationMoveResults() {
         metrics.getNumContainerMovesCompletedInLatestIteration());
   }
 
+  private long performTimeMoveCancelAndCount() {
```

Comment:
Let's rename this method to something like `cancelAndCountTimedOutMoves`?

Reply:
renamed as cancelAndCountPendingMoves as applicable for cancelling on exception case, not only timeout.